models/User.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
```

Comment:
Needs to be updated with the other User details. Here are the default values:
{
        "name" : user input string,
        "email": user input string,
        "password": hashed string,
        "properties" : {
                "profile_pic" : empty string (which can be parsed by the View to display a white box, or whatever we choose for default. This way, we save space in Mongo)
                "profile_desc" : empty string
                "posts" : empty array
                "favorites" : empty array
        },
        "account_info" : {
                "last_login" : empty string
        },
        "delete_flag" : boolean false // this represents whether the acc is considered deleted and therefore inaccessible
}